src/database/metadata.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Package for interacting with Realtime Database metadata.
+ */
+
+import * as api from "../api";
+import * as logger from "../logger";
+import * as utils from "../utils";
+
+function handleErrorResponse(response: any): any {
```

Comment:
Does this function return a promise?

Reply:
yes, apparently. tightened up the type signature